Reflections/Week_13_Reflection.md

```diff
@@ -1,22 +1,22 @@
 1. What have you accomplished this week? (Please list each accomplishment, and explain) 
-  - I was able to conncect the Linear Actuator to the currently built system. (D4.5 shows this.)
-  - I was able to get it to work together with only a few hickups, since they were built sepertly, so new parts needed to be intergrated or moved around. (D4.5 shows this.)
+  - I was able to connect the Linear Actuator to the currently built system. (D4.5 shows this.)
+  - I was able to get it to work together with only a few hiccups, since they were built separately, so new parts needed to be integrated or moved around. (D4.5 shows this.)
   - I was able to test and show off the system working in it's entirety. (D4.5 shows this.)
 2. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
-  - I had trouble figuring out where to move the Linear Actuator's motor since I couldn't have it hang off. And so i moved it to under the Linear Actuator, and right beside the Lego Brain. I achived this by transfering the rotational movement of the motor with a axle and gear setup. (D4.5 shows this.)
+  - I had trouble figuring out where to move the Linear Actuator's motor since I couldn't have it hang off. And so, I moved it to under the Linear Actuator, and right beside the Lego Brain. I achieved this by transferring the rotational movement of the motor with an axle and gear setup. (D4.5 shows this.)
```

Comment:
I watched the video and it's great that your system is able to change directions along with the weight. In you D4.5 can you specify how much weight you tested your system with? And how much the whole system weighs with the new changes?